content/_data/authors/ryansmith303.adoc

```diff
@@ -0,0 +1,6 @@
+---
+name: "Ryan Smith"
+github: ryansmith303
+---
+
+Ryan Smith is a Certified CloudBees Jenkins Platform Engineer, and currently works as a Senior Developer Support Engineer for CloudBees. Ryan currently leads the performance and stability team within the support organization, and spends his days evangelizing enterprise Java deployment best practices. He resides in Grand Junction, CO and would love to discuss JVM garbage collection over a cocktail with you.
```

Comment:
Since this is intended for an international audience, it may be best to include the country in the location information. 
```suggestion
Ryan Smith is a Certified CloudBees Jenkins Platform Engineer, and currently works as a Senior Developer Support Engineer for CloudBees. Ryan currently leads the performance and stability team within the support organization, and spends his days evangelizing enterprise Java deployment best practices. He resides in Grand Junction, Colorado, USA and would love to discuss JVM garbage collection over a cocktail with you.
```